docs/user/metrics/boolean.md

```diff
@@ -102,6 +102,27 @@ assert True is metrics.flags.a11y_enabled.test_get_value()
 
 </div>
 
+<div data-lang="C#" class="tab">
+
+```C#
+using static Mozilla.YourApplication.GleanMetrics.FlagsOuter;
```

Comment:
`FlagsOuter`?

Reply:
Yes, that's an artefact of the way C# works, due to some limitations. This might change when implementing code generation, but that's the way it is.